### PR TITLE
Feature/update release notes

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- R17 release notes and summary

### CODE REVIEW

**Comment by emclaug2** [comment]
Can someone provide a link to where users can find the new PX Blue components in Figma?

**Comment by huayunh** [comment]
Figma sticker sheet needs to be (1) published first and then (2) republished into a public file for ppl to access outside of Eaton 

The link should be the same — should still be that community plug in thing

File: src/docs/release-notes/R17/Summary.mdx [hunk 1]
```diff
@@ -0,0 +1 @@
+Updated our design pattern pages with modern images and restructured each page's content. Expanded the PX Blue stickersheet to include more PX Blue and Material UI components. Added reusable authentication workflow for React. Added new components for Angular (Nav Rail, Dropdown Toolbar) & React (Dropdown Toolbar), and React Native (Spacer, ListItemTag).
```
Comment (joebochill) [inline src/docs/release-notes/R17/Summary.mdx:1]:
```suggestion
Updated content for the design pattern pages, added more components to the PX Blue stickersheet, new Angular components for [Dropdown Toolbar](https://pxblue-components.github.io/angular/?path=/info/components-dropdown-toolbar--readme) and [Navigation Rail](https://pxblue-components.github.io/angular/?path=/story/components-drawer--with-a-nav-rail), new React component for [Dropdown Toolbar](https://pxblue-components.github.io/react/?path=/info/components-dropdown-toolbar--get-read-me-story), new React Native components for [Spacer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--spacer) and [List Item Tag](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--list-item-tag), built a re-usable authentication workflow for React.
```

File: src/docs/release-notes/R17/R17.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+-   Added more images and gifs to showcase each design pattern.
+-   Reorganized each design pattern page to have a behaviors, anatomy, specifications, and developer section.
+-   Completed migration of [Angular](https://github.com/pxblue/angular-design-patterns), [Ionic](https://github.com/pxblue/ionic-design-patterns), and [React Native](https://github.com/pxblue/react-native-design-patterns) design patterns into a mono-repo.
+
+### Component Libraries
+
+-   Added [Spacer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--spacer) and [List Item Tag](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--list-item-tag) components to React Native
+-   Added [Dropdown Toolbar](https://pxblue-components.github.io/react/?path=/info/components-dropdown-toolbar--get-read-me-story) component to React
+-   Added [Dropdown Toolbar](https://pxblue-components.github.io/angular/?path=/info/components-dropdown-toolbar--readme) component and Nav-Rail component to Angular
```
Comment (joebochill) [inline src/docs/release-notes/R17/R17.mdx:16]: Follow the same (component libraries) format we used for R16

File: src/docs/release-notes/R17/R17.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+### Stickersheet Components
+
+-   Added PX Blue components (EmptyState, ChannelValue, Heroes, ListItemTag, InfoListItem, NavRail)
+-   Added more Material UI components (Bottom Sheets, Chips, Snackbars, Cards, Sliders)
+
+### Design Patterns
+
+-   Added more images and gifs to showcase each design pattern.
+-   Reorganized each design pattern page to have a behaviors, anatomy, specifications, and developer section.
```
Comment (joebochill) [inline src/docs/release-notes/R17/R17.mdx:9]:
```suggestion
-   Re-organized and added new content to explain the patterns
-   Added more images and design specifications.
```

File: src/docs/release-notes/R17/R17.mdx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+### Component Libraries
+
+-   Added [Spacer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--spacer) and [List Item Tag](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--list-item-tag) components to React Native
+-   Added [Dropdown Toolbar](https://pxblue-components.github.io/react/?path=/info/components-dropdown-toolbar--get-read-me-story) component to React
+-   Added [Dropdown Toolbar](https://pxblue-components.github.io/angular/?path=/info/components-dropdown-toolbar--readme) component and Nav-Rail component to Angular
+
+### Reusable Workflows
+
+-   Reusable [authentication workflow](https://github.com/pxblue/react-workflows) for React.
```
Comment (joebochill) [inline src/docs/release-notes/R17/R17.mdx:20]:
```suggestion
-   Reusable [Authentication Workflow](https://www.npmjs.com/package/@pxblue/react-auth-workflow) for React.
```

File: src/docs/release-notes/R17/R17.mdx [hunk 5]
```diff
@@ -0,0 +1,20 @@
+### Stickersheet Components
+
+-   Added PX Blue components (EmptyState, ChannelValue, Heroes, ListItemTag, InfoListItem, NavRail)
+-   Added more Material UI components (Bottom Sheets, Chips, Snackbars, Cards, Sliders)
+
+### Design Patterns
+
+-   Added more images and gifs to showcase each design pattern.
+-   Reorganized each design pattern page to have a behaviors, anatomy, specifications, and developer section.
+-   Completed migration of [Angular](https://github.com/pxblue/angular-design-patterns), [Ionic](https://github.com/pxblue/ionic-design-patterns), and [React Native](https://github.com/pxblue/react-native-design-patterns) design patterns into a mono-repo.
```
Comment (joebochill) [inline src/docs/release-notes/R17/R17.mdx:10]:
```suggestion
-   Completed migration of [Angular](https://github.com/pxblue/angular-design-patterns), [Ionic](https://github.com/pxblue/ionic-design-patterns), and [React Native](https://github.com/pxblue/react-native-design-patterns) design patterns into a single example repository.
```

**Review by joebochill — COMMENTED** [review]
We're also going to have new doc-site content around  Accessibility, Designing for Websites, and Illustrations.

Right to left support in component libraries

Miscellaneous Updates:
- Bug fixes
- Updated storybooks?
- New progress Icon

File: src/docs/release-notes/R17/R17.mdx (hunk 2 above)
Comment (emclaug2) [reply src/docs/release-notes/R17/R17.mdx:16]: I updated the format to match what was done for R16, also added a note about progress icon labels and the new UPS icon.

Comment (emclaug2) [reply src/docs/release-notes/R17/R17.mdx:16]: Forgot that we have a storybook addon for RTL support as well.

File: src/docs/release-notes/R17/R17.mdx [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+-   Added PX Blue components (EmptyState, ChannelValue, Heroes, ListItemTag, InfoListItem, NavRail)
+-   Added more Material UI components (Bottom Sheets, Chips, Snackbars, Cards, Sliders)
+
+### Design Patterns
+
+-   Re-organized and added new content to explain the patterns
+-   Added more images and design specifications.
+-   Completed migration of [Angular](https://github.com/pxblue/angular-design-patterns), [Ionic](https://github.com/pxblue/ionic-design-patterns), and [React Native](https://github.com/pxblue/react-native-design-patterns) design patterns into a single example repository.
+-   Added documentation around accessibility standards and how to design websites and illustrations.
```
Comment (joebochill) [inline src/docs/release-notes/R17/R17.mdx:11]: I would either pop this out into a separate section around Documentation Site, or stick it under miscellaneous (these aren't technically design patterns)

Comment (emclaug2) [reply src/docs/release-notes/R17/R17.mdx:11]: I think this makes sense; are there public links to these resources we can place here?

File: src/docs/release-notes/R17/R17.mdx [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    -   For Angular, we now have components for [Dropdown Toolbar](https://pxblue-components.github.io/angular/?path=/info/components-dropdown-toolbar--readme) and added a [nav-rail variant](https://pxblue-components.github.io/angular/?path=/story/components-drawer--with-a-nav-rail) to our Drawer component.
+    -   For React, we now have components for [Dropdown Toolbar](https://pxblue-components.github.io/react/?path=/info/components-dropdown-toolbar--readme).
+    -   For React Native, we now components for [Spacer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--spacer) and [List Item Tag](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--list-item-tag).
+-   Enhanced our component libraries to support right-to-left languages and developed/integrated a [RTL toggle](https://www.npmjs.com/package/@pxblue/storybook-rtl-addon) into our storybook demos.
+-   Added a new progress icon for UPS indicators and added optional labels to all progress icons.
```
Comment (joebochill) [inline src/docs/release-notes/R17/R17.mdx:20]: Another one I would pop under miscellaneous (we also added / will be adding a couple new lighting-related icons)

File: src/docs/release-notes/R17/R17.mdx (hunk 6 above)
Comment (joebochill) [reply src/docs/release-notes/R17/R17.mdx:11]: Jeff is building the first two right now, but you can add the links to them `/design/accessibility` and `/design/websites` and the other one is `style/illustrations`

File: src/docs/release-notes/R17/R17.mdx [hunk 8]
```diff
@@ -5,8 +5,8 @@
 
 ### Design Patterns
 
--   Added more images and gifs to showcase each design pattern.
--   Reorganized each design pattern page to have a behaviors, anatomy, specifications, and developer section.
+-   Re-organized and added new content to explain the patterns
```
Comment (huayunh) [inline src/docs/release-notes/R17/R17.mdx:8]:
```suggestion
-   Re-organized and added new content to explain our [design patterns](/patterns).
```

File: src/docs/release-notes/R17/Summary.mdx [hunk 9]
```diff
@@ -0,0 +1 @@
+Updated content for the design pattern pages, added more components to the PX Blue stickersheet, new Angular components for [Dropdown Toolbar](https://pxblue-components.github.io/angular/?path=/info/components-dropdown-toolbar--readme) and [Navigation Rail](https://pxblue-components.github.io/angular/?path=/story/components-drawer--with-a-nav-rail), new React component for [Dropdown Toolbar](https://pxblue-components.github.io/react/?path=/info/components-dropdown-toolbar--get-read-me-story), new React Native components for [Spacer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--spacer) and [List Item Tag](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--list-item-tag), built a re-usable authentication workflow for React.
```
Comment (huayunh) [inline src/docs/release-notes/R17/Summary.mdx:1]: add a link to patterns page here too

File: src/docs/release-notes/R17/R17.mdx (hunk 6 above)
Comment (emclaug2) [reply src/docs/release-notes/R17/R17.mdx:11]: Got it, thank you 👍

File: src/docs/release-notes/R17/Summary.mdx (hunk 9 above)
Comment (emclaug2) [reply src/docs/release-notes/R17/Summary.mdx:1]: Added a link out to the design patterns page.

File: src/docs/release-notes/R17/R17.mdx (hunk 7 above)
Comment (emclaug2) [reply src/docs/release-notes/R17/R17.mdx]: I added a new misc section and moved the progress icons to be the first item here.

File: src/docs/release-notes/R17/R17.mdx [hunk 10]
```diff
@@ -0,0 +1,30 @@
+### Stickersheet Components
+
+-   Added PX Blue components (EmptyState, ChannelValue, Heroes, ListItemTag, InfoListItem, NavRail)
```
Comment (huayunh) [inline src/docs/release-notes/R17/R17.mdx:3]: Kyle is working on all the other pxb drawers at the moment. Should we include that?

Comment (huayunh) [reply src/docs/release-notes/R17/R17.mdx:3]: Kyle Harris is working on user menu

File: src/docs/release-notes/R17/R17.mdx [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+-   Added more Material UI components (Bottom Sheets, Chips, Snackbars, Cards, Sliders)
```
Comment (huayunh) [inline src/docs/release-notes/R17/R17.mdx:4]: Kyle Harris is working on tooltips

File: src/docs/release-notes/R17/R17.mdx (hunk 10 above)
Comment (huayunh) [reply src/docs/release-notes/R17/R17.mdx:3]: And scorecard should be here too

File: src/docs/release-notes/R17/R17.mdx [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+
+### Design Patterns
+
+-   Re-organized and added new content to explain our [design patterns](/patterns).
+-   Added more images and design specifications.
+-   Completed migration of [Angular](https://github.com/pxblue/angular-design-patterns), [Ionic](https://github.com/pxblue/ionic-design-patterns), and [React Native](https://github.com/pxblue/react-native-design-patterns) design patterns into a single example repository.
+
+### Documentation Site
+
+-   Added documentation around [accessibility standards](https://pxblue.github.io/design/accessibility) and how to design [websites](https://pxblue.github.io/design/websites) and [illustrations](https://pxblue.github.io/style/illustrations).
```
Comment (huayunh) [inline src/docs/release-notes/R17/R17.mdx:14]: There are two and's in this sentence

File: src/docs/release-notes/R17/R17.mdx [hunk 13]
```diff
@@ -0,0 +1,30 @@
...
+    -   For React Native, we now components for [Spacer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--spacer) and [List Item Tag](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--list-item-tag).
+-   Enhanced our component libraries to support right-to-left languages and developed/integrated a [RTL toggle](https://www.npmjs.com/package/@pxblue/storybook-rtl-addon) into our storybook demos.
+
+### Reusable Workflows
+
+-   Reusable [Authentication Workflow](https://www.npmjs.com/package/@pxblue/react-auth-workflow) for React.
+
+### Miscellaneous
+
+-   Added new progress icons and an optional percentage label.
```
Comment (huayunh) [inline src/docs/release-notes/R17/R17.mdx:30]: Link to npm or github

File: src/docs/release-notes/R17/R17.mdx (hunk 10 above)
Comment (emclaug2) [reply src/docs/release-notes/R17/R17.mdx:3]: I'll include the Drawers because we'll probably get it done before R17 and include all others.

Comment (joebochill) [reply src/docs/release-notes/R17/R17.mdx:3]: Yep, we should have these ready for R17

File: src/docs/release-notes/R17/R17.mdx (hunk 11 above)
Comment (emclaug2) [reply src/docs/release-notes/R17/R17.mdx:4]: Added tooltips

**Comment by emclaug2** [comment]
Circle CI is failing because the release notes have links to resources that aren't published yet.

image

**Comment by joebochill** [comment]
> Circle CI is failing because the release notes have links to resources that aren't published yet.

If you change them to relative links and merge the latest from dev, they should pass

File: src/docs/release-notes/R17/R17.mdx [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+
+### Design Patterns
+
+-   Re-organized and added new content to explain our [design patterns](/patterns).
+-   Added more images and design specifications.
+-   Completed migration of [Angular](https://github.com/pxblue/angular-design-patterns), [Ionic](https://github.com/pxblue/ionic-design-patterns), and [React Native](https://github.com/pxblue/react-native-design-patterns) design patterns into a single example repository.
+
+### Documentation Site
+
+-   Added documentation around [accessibility standards](https://pxblue.github.io/design/accessibility) and how to design [websites](https://pxblue.github.io/design/websites) and [illustrations](https://pxblue.github.io/style/illustrations).
```
Comment (joebochill) [inline src/docs/release-notes/R17/R17.mdx:14]:
```suggestion
-   Added documentation around [accessibility standards](/design/accessibility), how to design [websites](/design/websites), and [illustrations](/style/illustrations).
```

File: src/docs/release-notes/R17/R17.mdx [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+### Component Libraries
+
+-   Added new components to our [Angular](https://pxblue-components.github.io/angular), [React](https://pxblue-components.github.io/react), and [React Native](https://pxblue-components.github.io/react-native) component libraries:
+    -   For Angular, we now have components for [Dropdown Toolbar](https://pxblue-components.github.io/angular/?path=/info/components-dropdown-toolbar--readme) and added a [nav-rail variant](https://pxblue-components.github.io/angular/?path=/story/components-drawer--with-a-nav-rail) to our Drawer component.
+    -   For React, we now have components for [Dropdown Toolbar](https://pxblue-components.github.io/react/?path=/info/components-dropdown-toolbar--readme).
+    -   For React Native, we now components for [Spacer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--spacer) and [List Item Tag](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--list-item-tag).
+-   Enhanced our component libraries to support right-to-left languages and developed/integrated a [RTL toggle](https://www.npmjs.com/package/@pxblue/storybook-rtl-addon) into our storybook demos.
```
Comment (joebochill) [inline src/docs/release-notes/R17/R17.mdx:22]:
```suggestion
-   Enhanced our component libraries to support right-to-left languages and developed / integrated a [RTL toggle](https://www.npmjs.com/package/@pxblue/storybook-rtl-addon) into our storybook demos.
```

**Review by joebochill — APPROVED** [review]

**Comment by emclaug2** [comment]
Thanks @joebochill , going to test out the relative links now.

**Pull request merged**